docs/reference/server-configuration.md

```diff
@@ -145,19 +167,23 @@ For more information, see
 
 ## `ksql.functions.substring.legacy.args`
 
-**Per query:** yes
```

Comment:
@rodesai not sure about this one but I think we actually need to just remove it from the docs entirely. I couldn't even find a reference to it in any of the very old ksql branches

Reply:
removed